src/content/pages/about/index.md

```diff
@@ -61,6 +55,6 @@ PyCon AU is typically held in two year blocks at the same city, with some variat
 - [2020](https://2020.pycon.org.au), [2021](https://2021.pycon.org.au): The Internet
 - (PyCon AU was not held in 2022)
 - [2023](https://2023.pycon.org.au): Tarntanya/Adelaide, SA
-- 2024: To be announced
+- 2024: Narrm/Melbourne, VIC
```

Comment:
Probably worth adding the link to the 2024 site now

Reply:
Good call.